node/hyperbahn/handler.js

```diff
@@ -296,3 +324,8 @@ HyperbahnHandler.prototype.advertise =
 function advertise(service) {
     throw new Error('not implemented');
 };
+
+HyperbahnHandler.prototype.unadvertise =
```

Comment:
This should work:
self.channel.topChannel.handler.removeServicePeer(service.serviceName, service.hostPort);